core/src/view/view.cpp

```diff
@@ -134,7 +135,12 @@ void View::setRoll(float _roll) {
 
 void View::setPitch(float _pitch) {
 
-    m_pitch = glm::clamp(_pitch, 0.f, (float)HALF_PI);
+    float max_pitch = HALF_PI;
+    if (m_type != CameraType::perspective) {
+        // Prevent projection plane from intersecting ground plane
+        max_pitch = atan2(m_pos.z, m_height * .5f);
```

Comment:
Why is `m_height` the correct thing to check against in landscape view? I would believe it should be `m_width` right when in landscape mode?.. because you will be doing the shove gesture with respect to that orientation?